src/main/java/com/example/democamel/helper/ParseDateTimeFromFile.java

```diff
@@ -0,0 +1,15 @@
+package com.example.democamel.helper;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class ParseDateTimeFromFile {
+
+    public LocalDateTime parse(String dateString) {
```

Comment:
Why do you need to manipulate strings, instead of using a different time format pattern?